components/folder/Header/Header.tsx

```diff
@@ -1,16 +1,18 @@
 import { useState, ChangeEvent } from "react";
-import style from "./header.module.css";
-import FolderModal from "components/Modal/FolderModal/FolderModal";
-import linkIcon from "assets/images/ic_link.svg";
-import useModal from "utils/hooks/useModal";
-import { ADD_LINK } from "utils/constants/strings";
-import { GetFolderResponse } from "types/apis";
+import Image from "next/image";
+import styles from "./header.module.css";
+import FolderModal from "@/components/common/Modal/FolderModal/FolderModal";
+import useModal from "hooks/useModal";
+import { ADD_LINK } from "constants/strings";
+import clsx from "clsx";
+import type { Folder } from "hooks/useGetFolders";
 
 interface Props {
-  list: GetFolderResponse[] | null;
+  list: Folder[] | null;
```

Comment:
이렇게 데이터를 명확하게 표현해주는게 더 좋을 것 같아요!
```suggestion
  folderList: Folder[] | null;
```